src/components/Image/Image.js

```diff
@@ -33,15 +33,27 @@ const LazyLoadedImage = (props) => {
     // the low quality image (props.image.thumbnail) will be hidden
   };
 
+  // This block checks whether the user specified fallback image actually works or not
+  const [isLoading, setIsLoading] = useState(true);
+  const [isValidFallback, setIsValidFallback] = useState(false);
+  useEffect(() => {
+    fetch(props.placeholderImg).then(res => {
```

Comment:
Does this implementation support local file paths?

Reply:
Tested it with a local file and it should support them.